gatsby-node.js

```diff
@@ -21,12 +22,26 @@ exports.createPages = async ({ graphql, actions, reporter }) => {
     }
   `);
 
-  if (result.errors) {
+  const wordpressResult = await graphql(`
+    {
+      allWpPost {
+        edges {
+          node {
+            slug
+          }
+        }
+      }
+    }  
+  `);
+
+
+  if (result.errors || wordpressResult.errors) {
     reporter.panicOnBuild("Error while running GraphQL query.");
     return;
   }
 
   const posts = result.data.postsRemark.edges;
+  const wordpressPosts = wordpressResult.data.allWpPost.edges;
```

Comment:
can we just do something like this just to be sure `wordpressResult?.data?.allWpPost?.edges || [];`

Reply:
Yes, we can. Made a new commit with that.